tilequeue/process.py

```diff
@@ -581,12 +581,9 @@ def convert_source_data_to_feature_layers(rows, layer_data, bounds, zoom):
     for layer_datum in layer_data:
         layer_name = layer_datum['name']
         features = features_by_layer[layer_name]
-        # TODO padded bounds
-        padded_bounds = dict(
-            polygon=bounds,
-            line=bounds,
-            point=bounds,
-        )
+        query_bounds_pad_fn = layer_datum['query_bounds_pad_fn']
```

Comment:
@iandees this is part 2 of the change.  Rather than just using the unpadded_bounds passed in, use the query_bounds_pad_fn attached to the layer datum to create an appropriately padded bbox.